inst/modeldb/therapeuticArea/oncology/tgi_no_sat_expo.R

```diff
@@ -14,15 +15,23 @@ tgi_no_sat_expo <- function() {
     ts0 <- exp(lts0)
     ka <- exp(lka)
     cl <- exp(lcl)
-    vc  <- exp(lvc)
+    vc <- exp(lvc)
     kge <- exp(lkge)
+    kgl <- exp(lkgl)
     
     kel <- cl / vc
     tumorSize(0) <- ts0
+    depot(0) <- 0
```

Comment:
Please remove the initial condition setting to zero since it's implied (this line and the next).